src/dec_png.c

```diff
@@ -148,7 +176,9 @@ int SUB_NAME(unsigned char *pngbuf,g2int *width,g2int *height,char *cout)
       }
     }
 
-/*      Clean up   */
+/*
+**     Clean up
+*/
```

Comment:
For the rest of this doxygenation task, please put up each file as a separate PR, to make them easier to review, and please don't reformat the comments in the code. If you want to do that in the future, you can, but we can discuss what the format should be. For now, let's just focus on doxygenation.

Reply:
Okay